src/activity-handlers/create.handler.ts

```
@@ -24,10 +24,23 @@ export class CreateHandler {
 
-        const sender = await create.getActor(ctx);
-        if (sender === null || sender.id === null) {
-            ctx.data.logger.info('Create sender missing, exit early');
-            return;
+        try {
+            const sender = await create.getActor(ctx);
+            if (sender === null || sender.id === null) {
+                ctx.data.logger.info('Create sender missing, exit early');
+                return;
+            }
+        } catch (err: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)